indexer/entity/src/dex_mean_price.rs

```diff
@@ -0,0 +1,65 @@
+use sea_orm::entity::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Deserialize, Serialize)]
+#[sea_orm(table_name = "DexMeanPrice")]
+pub struct Model {
```

Comment:
I wonder if we should track which DEX was used as the source for a price point since we wanted to add more DEXs as well

Reply:
This can be determined by the `address_id` value (pool address, specific for each DEX, and sometimes even for a specific asset pair). Do you think it would be useful also to explicitly store an enum value with the DEX task identificator that created the price point?

Reply:
hmm, fair point. It would mean the webserver needs to have a registry of DEXs to address IDs though

Reply:
> the webserver needs to have a registry of DEXs to address IDs though

Yes, and that might actually be quite problematic, since the addresses are not fixed (e.g. they depend on the LP's current stake pool choice for ADA-token LPs). What about adding a dex column to the mean price table and then querying based on that instead of the address? Like this https://github.com/dcSpark/carp/pull/119/commits/ad56d0bae2522b1c9c72a53a7e6634c31a23ac79